elements/document/nuxeo-document-import.js

```diff
@@ -351,6 +351,21 @@ Polymer({
       paper-progress {
         margin-top: 8px;
       }
+
+      button.link {
+        color: var(--nuxeo-secondary-color, #0066ff);
+        text-decoration: underline;
```

Comment:
We should use the `nuxeo-link-color` and remove the underline to match the style we have currently.
```suggestion
        color: var(--nuxeo-link-color, ##3a3a54);
```
Original:
![Screenshot 2021-12-07 at 10 22 04](https://user-images.githubusercontent.com/1880896/145013105-81aa7e04-8e5f-434d-9f68-b576d4d59fda.png)

Change:
![Screenshot 2021-12-07 at 10 21 56](https://user-images.githubusercontent.com/1880896/145013145-9d955c5c-8239-44da-ab5c-3fd0791fae78.png)

Reply:
Note that this means changing the way it looks today, and I would consider this outside the scope of this ticket, which is about fixing CSP issues. I'd rather have a separate ticket open to address this.

Reply:
Sorry, you are right. I misunderstood the original vs change screenshots. Will do the change, thanks.